src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -49,13 +49,35 @@ def _post(self, request: Request):
             return Response("Relay unauthorized for full config information", 403)
 
         version = request.GET.get("version") or "1"
+
         set_tag("relay_protocol_version", version)
 
-        if self._should_use_v3(version, request):
+        if version == "4":
+            # With v4, we want to move 'measurements' and 'metricsConditionalTagging' into a new
+            # global config.
```

Comment:
This comment will become stale when we modify the config, what do you think if we remove it?